src/Chainweb/BlockHeaderDB.hs

```diff
@@ -24,7 +24,7 @@ module Chainweb.BlockHeaderDB
 (
 -- * Chain Database Handle
   Configuration(..)
-, BlockHeaderDb
+, BlockHeaderDb(..)
```

Comment:
Almost certainly naughty. I imagine that the constructor was hidden for a reason.

Reply:
I just took a look, if there is a reason its not obvious, but Pact needs the three types contained within.

Reply:
@larskuhtz ?